tests/system/common.py

```diff
@@ -83,3 +85,43 @@ def wait_for_mesos_endpoint(timeout_sec=timedelta(minutes=5).total_seconds()):
     it returns false"""
 
     return shakedown.time_wait(lambda: shakedown.mesos_available_predicate(), timeout_seconds=timeout_sec)
+
+
+def metronome_available_predicate():
+    url = metronome_api_url()
+    try:
+        response = http.get(url)
+        return response.status_code == 200
+    except Exception as e:
+        return False
+
+
+def metronome_api_url():
+    return shakedown.dcos_url_path("/service/metronome/v1/jobs")
+
+
+def assert_job_run(client, job_id, runs_number=1, tasks_number=1):
+    job_runs_count = len(client.get_runs(job_id))
+    assert job_runs_count == runs_number, "Expecting 1 job run but found {} for job {}.".format(job_runs_count, job_id)
```

Comment:
Not sure why my previous comment was marked as outdated. Anyway `s/Expecting 1/Expecting ${runs_number}`